composer.json

```diff
@@ -25,12 +25,14 @@
         "symfony/yaml": "^5.4 || ^6.3 || ^7.0"
     },
     "require-dev": {
-        "doctrine/annotations": "^1.8.0 || ^2.0",
+        "doctrine/annotations": "^1.13.1 || ^2.0",
         "doctrine/data-fixtures": "^1.7",
+        "doctrine/dbal": "^3.7",
         "doctrine/doctrine-bundle": "^2.11",
         "doctrine/doctrine-fixtures-bundle": "^3.5.1 || ^4.0",
+        "doctrine/mongodb-odm": "^2.2",
         "doctrine/mongodb-odm-bundle": "^4.2 || ^5.0",
-        "doctrine/orm": "^2.7",
+        "doctrine/orm": "^2.14",
```

Comment:
The various Doctrine packages and the DoctrineBundle all require ORM 2.14 or later, so this just matches up.